### PR TITLE
Upgraded to rdc 1.0.3 and fixed breaking errors

### PR DESCRIPTION
**Pull request by Hoidi**
Thanks @molleer

### CODE REVIEW

**Comment by molleer** [comment]
No problem 👉😎👉

**Comment by Hoidi** [comment]
Saknar digIT merge privilegier :(

**Comment by molleer** [comment]
The markdown does not seem to work properly
bild
@Hoidi  Do you still think this is ok, or should this be fixed first?
@Portals Any idea what is wrong with the `DigitMarkdown` component?

**Comment by Hoidi** [comment]
> The markdown does not seem to work properly

Thought it was just that I didn't have the right fonts or something :(

**Comment by Portals** [comment]
DigitMarkdown is a bit of a mess. Please create an issue on the RDC repo.

File: frontend/package.json [hunk 1]
```diff
@@ -3,15 +3,16 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "font-awesome": "4.7.0",
-        "@cthit/react-digit-components": "^1.0.0-beta.42",
         "@babel/runtime": "7.5.4",
+        "@cthit/react-digit-components": "^1.0.3",
```
Comment (Portals) [inline frontend/package.json:6]: You need to update the dependencies that RDC v1.0.3 needs.

For example, redux is needed at all.

File: frontend/src/components/elements/tutorial/index.jsx [hunk 2]
```diff
@@ -7,58 +7,51 @@ import {
...
+    })
+    return (
+        <DigitDesign.Card size={{width:"400px", height:"200px"}}>
+            <DigitDesign.CardBody
+                style={{cursor: 'pointer'}}
+                onClick={() => {
+                    openDialog({
+                        renderButtons: () => null,
+                        renderMain: () => (
+                            <div>
```
Comment (Portals) [inline frontend/src/components/elements/tutorial/index.jsx]: Unnecessary div?

Comment (Portals) [reply frontend/src/components/elements/tutorial/index.jsx]: A fragment can be used instead.

**Review by Portals — CHANGES_REQUESTED** [review]

**Review by molleer — APPROVED** [review]
I think it is ok. Better that it looks bad that there being wrong information.

**Comment by Portals** [comment]
Still incorrect dependencies:
https://github.com/cthit/react-digit-components/releases/tag/v1.0.3

**Review by Portals — APPROVED** [review]

**Pull request merged**